tests/attribute_annotations.pyi

```diff
@@ -9,7 +8,6 @@ field7 = b""  # Y015 Attribute must not have a default value other than "..."
 class Foo:
     field1: int
     field2: int = ...
-    field3 = ...  # type: int
```

Comment:
If this triggered Y026, might be worth special casing `...`?

Reply:
Yes, I can probably just exclude cases where the RHS is ellipsis. We should lint against this but Y026 isn't the right code.